src/civetweb.c

```diff
@@ -14345,6 +14345,36 @@ set_ssl_option(struct mg_context *ctx)
 		return 1;
 	}
 
+	/* Check for external SSL_CTX */
+	SSL_CTX* ssl_ctx = 0;
+	callback_ret =
+	    (ctx->callbacks.get_external_ssl_ctx == NULL)
+	        ? 0
+	        : (ctx->callbacks.get_external_ssl_ctx(&ssl_ctx, ctx->user_data));
```

Comment:
Travis CI build error:
```
/home/travis/build/civetweb/civetweb/src/civetweb.c:14353:49: error: incompatible pointer types passing 'SSL_CTX **' (aka 'struct ssl_ctx_st **') to parameter of type 'void **' [-Werror,-Wincompatible-pointer-types]

                : (ctx->callbacks.get_external_ssl_ctx(&ssl_ctx, ctx->user_data));

                                                       ^~~~~~~~
```
==> `SSL_CTX* ssl_ctx = 0;` must be `void ** ssl_ctx = 0;` and `ctx->ssl_ctx = ssl_ctx;` needs a cast later.